### PR TITLE
Add fail on warning for RTD

### PR DESCRIPTION
**Pull request by decentral1se**
Same as our other configuration.

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ formats: all
 build:
   image: latest
 
+sphinx:
+  fail_on_warning: true
```
Comment (webknjaz) [inline .readthedocs.yml]: I believe this will be picked up from `setup.cfg`. So we probably don't need duplication...

Comment (decentral1se) [reply .readthedocs.yml]: Explicit better than blah blah blah blah blah ;)

Comment (webknjaz) [reply .readthedocs.yml]: Yes and no. The same thing duplicated in a lot of places makes it hard to track updates. Next time you'll update one place but not the other and it'll behave differently without any prompt why and where to look at.

It's a common knowledge that RTD invokes sphinx. And that sphinx has configs.
Sometimes implicit is better than explicit. You just need to be careful when making related decisions :)

Comment (decentral1se) [reply .readthedocs.yml]: Heresy!

**Pull request closed without merging**